resources/example_log_lines.ts

```diff
@@ -56,8 +85,58 @@ const exampleLogLines: ExampleLines = {
         '00|2021-04-26T14:12:30.0000000-04:00|302B||The gravity node uses Forked Lightning.|45d50c5f5322adf787db2bd00d85493d',
         '00|2021-04-26T14:12:30.0000000-04:00|322A||The attack misses.|f9f57724eb396a6a94232e9159175e8c',
         '00|2021-07-05T18:01:21.0000000-04:00|0044|Tsukuyomi|Oh...it\'s going to be a long night.|1a81d186fd4d19255f2e01a1694c7607',
+        '00|2020-02-26T18:59:23.0000000-08:00|0038||cactbot wipe|77364412c17033eb8c87dafe7ce3c665',
+        '00|2020-03-10T18:29:02.0000000-07:00|001D|Tini Poutini|Tini Poutini straightens her spectacles for you.|05ca458b4d400d1f878d3c420f962b99',
       ],
     },
+    unitTests: [
```

Comment:
I don't really agree with adding the unit test info to this file. This feels like a separation of responsibility issue, e.g. why should the built and deployed to users code include unit tests?

Maybe this requires a separate `test/helper/example_log_lines_test_map.ts` file or something.

Reply:
> I don't really agree with adding the unit test info to this file. This feels like a separation of responsibility issue, e.g. why should the built and deployed to users code include unit tests?

> Maybe this requires a separate test/helper/example_log_lines_test_map.ts file or something.

If the concern is including this in the deployed code, then the entire file could be moved to util/, since its only uses are for util/gen_log_guide and the unit tests.  The file isn't used as a runtime resource, so maybe that's better anyway.

I think I would prefer that approach over splitting the unit tests from the example lines because they closely related (same data, and connected via indexToTest), and I think it's helpful to be able to see and update example lines and the unit tests that use them in a single place. Putting them in different places means yet another file to update when log lines are added/changed + the need to keep the properties in sync.  And the unit tests need to read the example lines too.

Reply:
> I don't really agree with adding the unit test info to this file. This feels like a separation of responsibility issue, e.g. why should the built and deployed to users code include unit tests?

Been giving this further thought, and I'm okay with separating the unit tests from the example line data.  I think the case could be made either way for viewing this as test data that's exposed in LogGuide, or LogGuide-supporting data that we happen to also test against.  At the end of the day, regardless of implementation, Typescript will throw compile-time errors if log def types are added without both example lines and unit tests, and mocha will report mis-matched unit test data, and since those are the primary concerns, anything else is just convenience.

I am going to move the example lines to `util/` though, because there really isn't a need for it to be in `/resources`. I'm also going to separate out the test data and make the necessary changes in #139 rather than here, because there were additional structure changes there that make rebasing/resolving merge conflicts needlessly bloody.  (Yes, I could've avoided that by waiting to PR #139 until this was merged, but oh well.)